ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -1247,8 +1322,9 @@ gvl_ruby_callback(void* data) {
             (VALUE(*)(...))&rescue_callback, (VALUE)(&callback_data), rb_eException, (VALUE)0);
 
     if(callback_data.failed) {
+        /* TODO: Use the v8 exception as a data storage instead */
```

Comment:
Nit: If you are leaving a 'todo' in comments, then you should have an associated issue / work item reference. Otherwise it should change to a "NOTE: we can do a better job here if we do X."